packages/services/api/src/modules/operations/providers/clickhouse-client.ts

```diff
@@ -107,7 +107,8 @@ export class ClickHouse {
           lookup: 1000,
           connect: 1000,
           secureConnect: 1000,
-          request: timeout,
+          // override the provided timeout of a query with the globally configured timeout
+          request: this.config.requestTimeout ?? timeout,
```

Comment:
does this apply to the harcoded timeouts like this one:
https://github.com/kamilkisiela/graphql-hive/blob/main/packages/services/api/src/modules/operations/providers/operations-reader.ts#L694